oak_functions/loader/src/tests.rs

```diff
@@ -26,6 +26,8 @@ use std::net::{Ipv6Addr, SocketAddr};
 use test::Bencher;
 
 const TEST_WASM_MODULE_PATH: &str = "testdata/non-oak-minimal.wasm";
+const HELLO_WORLD_MODULE_PATH: &str = "testdata/hello_world.wasm";
```

Comment:
I could also use this in the `test_server` test. WDYT?

Reply:
Why does it get compiled to testdata? That seems odd, given it's not checked-in.

(edit): I just noticed it is checked in, that is not great, I think it should just compile it to some target / bin folder and pick it up from there, like we do for the rest of the examples.

Reply:
Thanks. Done.